app/views/moderations/_mod_sidebar_right.html.erb

```diff
@@ -0,0 +1,10 @@
+<div id="sidebar-wrapper-right" class="sidebar-wrapper sidebar-wrapper-right crayons-layout__sidebar-right">
+  <div id="quick-stat-card" class="crayons-card">
+    <div class="crayons-card__body mod-saying-hello">
+      <p class="fw-medium fs-2xl">Hello! 👋</p>
+      <p>
+        Thank you for helping to keep DEV safe! ❤️
```

Comment:
we should replace `DEV` with `ApplicationConfig["COMMUNITY_NAME"]`

Reply:
Will do - thanks!

Reply:
done! @rhymes